package.json

```diff
@@ -32,10 +32,8 @@
     "es2015/text.js"
   ],
   "scripts": {
-    "build": "babel src -d .; NODE_ENV=es2015 babel src -d ./es2015; node script/inject-sequences.js",
-    "clean": "rm -rf -- index.js text.js es2015",
```

Comment:
I just want `es2015/new-temp-file.js` to not exist after running `touch es2015/new-temp-file.js && npm run build`. If the current config does that, that’s 👍!